.travis.yml

```diff
@@ -1,14 +1,18 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
+
+os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
     - CONDA_PY=27
     - CONDA_PY=34
     - CONDA_PY=35
+    - CONDA_PY=36
```

Comment:
Thanks @jjhelmus . I think I bungled some in trying to fix it. Sorry. 😞 Was trying to hunt for the ref as to what was suppose to be broken. Will fix now.